client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -141,109 +140,105 @@ const MarketplaceThankYou = () => {
-							<img alt="yoast logo" width={ imageWidth } src={ yoastInstalledImage }></img>
-						</MarketplaceThankYouHeader>
-						<ThankYouBody>
-							<div>
-								<MarketplaceThankyouSection>
-									<MarketplaceHeaderTitle className="marketplace-thank-you__body-header wp-brand-font">
-										{ translate( 'Yoast SEO Premium is installed' ) }
-									</MarketplaceHeaderTitle>
-								</MarketplaceThankyouSection>
-								<MarketplaceThankyouSection>
-									<MarketplaceHeaderTitle
-										subtitle
-										className="marketplace-thank-you__body-header wp-brand-font"
-									>
-										{ translate( 'What’s next?' ) }
-									</MarketplaceHeaderTitle>
-									<MarketplaceNextSteps>
-										<h3>{ translate( 'Plugin setup' ) }</h3>
-										<div>
-											<p>
-												{ translate(
-													'Get to know Yoast SEO and customize it, so you can hit the ground running.'
-												) }
-											</p>
-											<div>
-												<FullWidthButton
-													href={ yoastSeoPageUrl }
-													primary
-													busy={ isRequestingMenu }
-													// TODO: Menu links are not properly loading on initial request, post transfer so yoastSeoPageUrl will remain empty post transfer
-													// This should be fixed with perhaps a work around to periodically poll for the menu with various domains until it loads
-													// or maybe blocking the user from entering this flow until a domain acquires SSL
-													disabled={ !! yoastSeoPageUrl }
-												>
-													{ translate( 'Get started' ) }
-												</FullWidthButton>
-											</div>
-										</div>
-										<h3>{ translate( 'Start putting SEO to work' ) }</h3>
-										<div>
-											<p>
-												{ translate(
-													"Improve your site's performance and rank higher with a few tips."
-												) }
-											</p>
-											<div>
-												<FullWidthButton
-													href={ postsPageUrl }
-													busy={ isRequestingMenu }
-													disabled={ !! yoastSeoPageUrl }
-												>
-													{ translate( 'View posts' ) }
-												</FullWidthButton>
-											</div>
-										</div>
-									</MarketplaceNextSteps>
-								</MarketplaceThankyouSection>
-								<MarketplaceThankyouSection>
-									<MarketplaceHeaderTitle
-										subtitle
-										className="marketplace-thank-you__body-header wp-brand-font"
-									>
-										{ translate( 'How can we help?' ) }
-									</MarketplaceHeaderTitle>
+			<Masterbar>
+				<Item
+					icon="cross"
+					onClick={ () =>
+						page( `/marketplace/product/details/wordpress-seo/${ selectedSiteSlug }` )
+					}
+					tooltip={ translate( 'Go to plugin' ) }
+					tipTarget="close"
+				/>
+			</Masterbar>
+			<MarketplaceThankYouContainer className="marketplace-thank-you__container checkout-thank-you">
+				<MarketplaceThankYouHeader>
+					<img alt="yoast logo" width={ imageWidth } src={ yoastInstalledImage }></img>
```

Comment:
It's so odd to have a closing `</img>` tag 😅

Reply:
An IDE autocompletion :D yup that looks wierd 🤣